articles/quickstart/native/react-native/_includes/_login.md

```diff
@@ -23,6 +25,8 @@ In the file `android/app/src/main/AndroidManifest.xml` you must make sure the **
 So your **MainActivity** should look like this:
 
 ```xml
+//android/app/src/main/java/com/auth0sample/MainActivity.java
```

Comment:
This is not correct, the `MainActivity` is a sub section inside the `//android/app/src/main/AndroidManifest.xml`